### DISCUSSION TITLE
大國的軍事科技發展：雷射、極音速和人工智慧

### DISCUSSION THREAD

**Discussion by Timmy110403540** [1101-GS3073-C/Python-Community-GS3073-C#168] [General]
對於雷射以及極音速武器，將不在這裡敘述。而熟悉的AI，其實也是可以當作武器。
於2011年後，因為大數據、機器學習等技術逐間成熟，AI不僅在商用領域發展，更在軍事領域有所創新。
舉幾個例子:如無人機、無人戰鬥裝甲車、無人水上載具等等機器。大多圍繞著「無人」，我想可能是因為只要操作機器即可，除了有著強大的破壞力外，還可以避免人員傷亡。
而在今年，陸軍無人戰鬥載具、海軍 USV 以及空軍 Skyborg 與空中母艦的發展，更是受到許多的關注。

文章來源:https://technews.tw/2021/12/30/reviewing-the-development-of-military-technoligy-in-2021-an-era-of-great-power-competition/

**Comment by minniewei** [comment]
雖然我對於科技發展十分看好，但會不會這些無人機器因為太過厲害反而傷害更多人(怕怕......)

**Reply by cobb110202008** [reply]
確實是一大憂患，戰爭對百姓造成的傷害太大了。

**Comment by bonny123865** [comment]
新聞裡的武器一個比一個看起來殺傷力更大，但還是希望不要有真正使用上的一天，會造成很多無辜的平民受傷。

**Comment by briankuo123** [comment]
常常看到電影會上演AI發展出自己的智慧並決定消滅人類，希望這件事情永遠不可能發生。

**Comment by wh20502** [comment]
我在想以後會不會有線上戰爭，給上面的戰爭模擬跑一跑就可以知道誰勝誰敗，後果跟成本如何，這樣可能以後才會贏來和平。

**Comment by jack90325** [comment]
科技的進步總是會伴隨戰爭的到來，這種高科技殺人武器希望是不要真的運用在戰爭上，不然人類就真的要毀滅了

**Comment by 110409538** [comment]
以後的世界可能會變成各個國家之間用科技來競爭。但我還是不希望國家之間用科技來進行戰爭。畢竟戰爭一定會有傷亡，只是嚴重程度的問題。

**Comment by ayakasanma** [comment]
人類被AI滅亡的科幻劇情可能將會成為一個嚴肅且沉重的話題了

**Comment by Shiba010** [comment]
難道以後發生戰爭，會變成是各國派出無人軍隊的較量嗎？！

**Comment by ghost** [comment]
這樣看來以後駭入敵方的無人機械或AI是更重要的吧
